README.md

```diff
@@ -1,6 +1,6 @@
 # Build@Mercari Training Program 2023
 
-This is @<your github id>'s build training repository.
+This is zhaomeiyimerukari's build training repository.
```

Comment:
This change has already been done in https://github.com/zhaomeiyimerukari/mercari-build-training-2023/pull/1.
In this kind of situation when the work is dependent upon the previous work/PR, please create a new branch from the branch of the dependent PR to avoid the redo. Here you can edit the origin branch name from `main` to `first-pull-request`.

Reply:
got it! THX!